lib/dndfonts.sty

```diff
@@ -22,6 +22,20 @@
       { \sffamily }
   }
 
+% Add the outlines
+  \NewDocumentCommand{\DndContour}{m}
+  {
+  \textpdfrender
```

Comment:
Please use the following here. It looks the closest without distorting the text.

\\\latex

  \NewDocumentCommand{\DndContour}{ o m }
  {
  \textpdfrender
    {
    TextRenderingMode = FillStroke,
    StrokeColor =       contourgray,
    LineWidth =         0.03ex,
    RenderingIntent =   RelativeColorimetric,
    #1,
    }{#2}
  }
```

Using round or bevel gives a nicer outline, but it distorts the filled text. I prefer the fill text to still have its normal font shape. The basic rules gets a sharp fill and rounded outline by using two different PDF objects. The outline is actually separate from the text.